core/Archive.php

```diff
@@ -789,21 +789,29 @@ private function prepareArchive(array $archiveGroups, Site $site, Period $period
         $invalidateBeforeArchiving = !SettingsServer::isArchivePhpTriggered();
 
         $parameters = new ArchiveProcessor\Parameters($site, $period, $this->params->getSegment());
+
+        $requestedReport = Common::getRequestVar('requestedReport', '', 'string');
+        if ($requestedReport) {
```

Comment:
should we accept this parameter only if `SettingsServer::isArchivePhpTriggered()`  as it means we know eg the super user is authenticated?

Reply:
👍 should have been that way, maybe I removed the code